pyproj/_transformer.pyx

```diff
@@ -202,6 +203,11 @@ cdef class _TransformerGroup:
                 operation_factory_context,
                 allow_ballpark,
             )
+            proj_operation_factory_context_set_discard_superseded(
+                self.context,
+                operation_factory_context,
+                int(not bool(allow_superseded)),
```

Comment:
I want PyProj's API to be opt-in to the superseded transformers. But since PROJ's API is opt-out, we need to invert the argument. I don't know enough about Cython to know if this is the right way to do this.